packages/pwa-kit-dev/src/utils/script-utils.test.js

```diff
@@ -134,6 +134,46 @@ describe('scriptUtils', () => {
         })
     })
 
+    describe('getLowestPackageVersion', () => {
```

Comment:
Add a test case for when the same package is installed with two different versions! And maybe one for two versions that are the same and have been deduped? I'm not sure what the tree looks like for that scenario.

Reply:
Oh, also add tests to ensure that the version comparison logic works! (e.g. `1.10.0 > 1.9.0`, `1.2.3 > 1.2.3-beta.0`)